controller/rancher/rancher_events_handler.go

```diff
@@ -0,0 +1,85 @@
+package rancher
+
+import (
+	"github.com/Sirupsen/logrus"
+	revents "github.com/rancher/event-subscriber/events"
+	"github.com/rancher/go-rancher/v2"
+)
+
+type EventsHandler interface {
+	Subscribe() error
+}
+
+type REventsHandler struct {
+	CattleURL       string
+	CattleAccessKey string
+	CattleSecretKey string
+}
+
+func (ehandler *REventsHandler) Subscribe() error {
+
+	eventHandlers := map[string]revents.EventHandler{
+		"target.drain": ehandler.handle,
+		"ping":         ehandler.handle,
+	}
+
+	router, err := revents.NewEventRouter("", 0, ehandler.CattleURL, ehandler.CattleAccessKey, ehandler.CattleSecretKey, nil, eventHandlers, "", 3, revents.DefaultPingConfig)
```

Comment:
Bump that 3 way up. We have it set to 250 in go-machine-service That should be a good number